src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -54,6 +59,70 @@ const PersonalDetails = (() => {
         $('#missing_details_notice').setVisibility(!!has_missing_field);
     };
 
+    const populateChangeableFields = () => {
+        if (!has_changeable_fields) return;
+
+        const landing_companies = State.getResponse('landing_company');
+        const changeable        = landing_companies.financial_company.changeable_fields;
+        if (changeable && changeable.only_before_auth) {
+            changeable_fields = changeable.only_before_auth;
+        }
+    };
+
+    /**
+     * Remove labels and static fields and replace them with input when fields are changeable.
+     *
+     * @param {get_settings} to prepopulate some of the values.
+     */
+    const displayChangeableFields = (get_settings) => {
+        if (!has_changeable_fields) return;
+        changeable_fields.forEach(field => {
+            CommonFunctions.getElementById(`row_${field}`).setVisibility(1);
+        });
+
+        if (changeable_fields.includes('first_name') || changeable_fields.includes('last_name') || changeable_fields.includes('salutation')) {
```

Comment:
would it be better to put these three in an array and then loop them? since adding a new field in the future would mean having to make it even longer